roles/setup_repo/tasks/PG_RedHat_rm_repos.yml

```diff
@@ -33,6 +33,7 @@
     - "/etc/yum.repos.d/edb.repo"
     - "/etc/yum.repos.d/edb.repo.rpmsave"
     - "/etc/yum.repos.d/enterprisedb-enterprise.repo"
+    - "/etc/yum.repos.d/enterprisedb-postgres_distributed.repo"
```

Comment:
Is the name of the repo `enterprisedb-postgres_distributed-x86_64.repo` and `enterprisedb-postgres_distributed-noarch.repo`? Is there way to do something like `enterprisedb-postgres_distributed-*.repo`?

Reply:
The yum.repos.d file is named enterprisedb-postgres_distributed.repo, and the entries in the file pertain to the x86_64 and noarch repositories.

Reply:
Awesome, thank you for confirming!